ansible/roles/run-test-target/tasks/test-collection-method.yml

```diff
@@ -127,6 +128,28 @@
         dest: "{{ logs_root }}/integration-test.log"
       delegate_to: localhost
 
+    - name: Move core dump files to core dump directory
+      become: yes
+      shell: |
+        mkdir /tmp/core_dumps
+        mv /tmp/core.out* /tmp/core_dumps || true
```

Comment:
Why do we need to move them?

Reply:
I couldn't find any other way of getting this to work. You can look at alternatives I tried in the other commits. When I kept the core.out* in the /tmp directory and loop over those files, it didn't work. I saw that the integration test log files were all in a directory and the logs were obtained by directory. I decided to do the same for the core dumps.

Reply:
I think this can be removed now, since we're writing them directly into `/tmp/core_dumps`